source/2019-12-20-the-ember-times-issue-129.md

```diff
@@ -0,0 +1,159 @@
+responsive: true
+---
+
+Happy Holidays, Emberistas! 🐹🌲⛄️
+
+Ember Octane is here 🐹🎉,
+new ember-autofocus-modifier 💡,
+3 new RFCs for template helpers 3️⃣,
+ember-apollo-client v2 released 🎉
+and join us for the last 11 days of DecEmber 1️⃣️⃣1️⃣!
+
+READMORE
+
+---
+
+## [Octane is Here 🎉](https://blog.emberjs.com/2019/12/20/octane-is-here.html)
+
+[Ember 3.15](https://blog.emberjs.com/2019/12/20/ember-3-15-released.html) was released this week. In addition to the usual new features, improvements and bug fixes, this release also marks the official release of Ember's first edition - **Ember Octane**!
+
+<!--alex ignore easy-->
+But what is Ember Octane? Ember Octane is the new, recommended way for developers to build Ember applications and addons.
+Octane features a new reactivity model, a renewed practice of building components and a stronger focus on HTML driven web development than ever before. Octane aims to make building Ember apps for developers not only more easy, but also **more fun** right from the start.
+And on top of that, Ember Octane is **fully opt-in** and **interoperable** for existing Ember apps - which means you can try out the new way of building modern Ember Octane apps gradually, without having to rewrite your entire app today.
+
+Want to know all about Ember Octane? Learn more about what's new in Octane and how to get started [in the official Ember Octane release announcement](https://blog.emberjs.com/2019/12/20/octane-is-here.html) by [Yehuda Katz (@wycats)](https://github.com/wycats)! Need help? Check out **#topic-octane-migration** channel on the [Ember Discord](https://discordapp.com/invite/emberjs)!
+
+---
+
+## [New ember-autofocus-modifier 💡](https://medium.com/qonto-engineering/autofocus-modifier-improving-our-component-apis-fd589b98f4e)
+
+While building an autofocus feature for their components, Qonto engineers were also able to improve their APIs. [Cyrille David (@dcyriller)](https://github.com/dcyriller) and team wanted to add autofocus to several components across two apps. For example, both their one-time-password input and their email input component needed autofocus. Since mixins are no longer recommended in the Ember ecosystem, Cyrille went with modifiers. [And they blogged about their experience here!](https://medium.com/qonto-engineering/autofocus-modifier-improving-our-component-apis-fd589b98f4e)
+
+Modifiers are a new feature in the upcoming Ember Octane edition for **sharing code between components**, similar to directives in Vue or Angular. We revisited [Chris Garrett (@pzuraq)](https://github.com/pzuraq)'s blog [Coming Soon in Ember Octane - Part 4: Modifiers](https://blog.emberjs.com/2019/03/06/coming-soon-in-ember-octane-part-4.html) and the [Octane Guides preview docs: Out-of-Component-Modifications](https://octane-guides-preview.emberjs.com/release/components/template-lifecycle-dom-and-modifiers/#toc_out-of-component-modifications) to learn more!
```

Comment:
Let's now remove the word "upcoming". :D

```suggestion
Modifiers are a new feature in the Ember Octane edition for **sharing code between components**, similar to directives in Vue or Angular. We revisited [Chris Garrett (@pzuraq)](https://github.com/pzuraq)'s blog [Coming Soon in Ember Octane - Part 4: Modifiers](https://blog.emberjs.com/2019/03/06/coming-soon-in-ember-octane-part-4.html) and the [Octane Guides preview docs: Out-of-Component-Modifications](https://octane-guides-preview.emberjs.com/release/components/template-lifecycle-dom-and-modifiers/#toc_out-of-component-modifications) to learn more!
```

Reply:
@ijlee2 Sounds good, I just updated it in a separate commit :tada: